package.json

```diff
@@ -10,7 +10,7 @@
     "test": "tap --100 -j1 -R classic test/index.test.js",
     "validate": "npm run lint && npm run test",
     "postgres": "docker run -p 5432:5432 --name fastify-slonik -e POSTGRES_PASSWORD=postgres -d postgres",
-    "load-data": "docker exec -it fastify-slonik psql -c 'CREATE TABLE users(id serial PRIMARY KEY, username VARCHAR (50) NOT NULL);' -U postgres -d postgres"
+    "load-data": "docker exec -i fastify-slonik psql -c 'CREATE TABLE users(id serial PRIMARY KEY, username VARCHAR (50) NOT NULL);' -U postgres -d postgres"
```

Comment:
Could we run the DB initialization with docker-compose?

Just move the following into a `fixtures/init.sql` file:
```sql
CREATE TABLE users(id serial PRIMARY KEY, username VARCHAR (50) NOT NULL);
```

and add the following to the `postgres` config in the docker-compose file:
```yml
volumes:
      - ./fixtures/init.sql:/docker-entrypoint-initdb.d/init.sql
```